compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -5807,7 +5811,12 @@ void t_java_generator::generate_java_scheme_lookup(ostream& out) {
 void t_java_generator::generate_javax_generated_annotation(ostream& out) {
   time_t seconds = time(nullptr);
   struct tm* now = localtime(&seconds);
-  indent(out) << "@javax.annotation.Generated(value = \"" << autogen_summary() << "\"";
+  if (!javax_annotations_) {
+    indent(out) << "@jakarta.annotation.Generated(value = \"" << autogen_summary() << "\"";
+  } else {
+    indent(out) << "@javax.annotation.Generated(value = \"" << autogen_summary() << "\"";
+  }
+
```

Comment:
can you swap two legs of the if statement?

Reply:
I modified the if statement